src/components/MintPreview/MintPreview.tsx

```diff
@@ -90,29 +100,23 @@ const MintPreview = (props: MintPreviewProps) => {
 							<p>{nft.story}</p>
 
 							<div className={styles.Container}>
-								{!exists && (
-									<div className={styles.IconContainer}>
-										<img alt={ALT_TEXT.QUESTION_MARK} src={questionMark} />
-									</div>
-								)}
+								<div className={styles.ButtonContainer}>
+									{nft.transactionHash.length > 0 && (
+										<FutureButton glow onClick={openProfile}>
+											{BUTTON_TEXT.VIEW_PROFILE}
+										</FutureButton>
+									)}
+								</div>
 								<div
 									className={`${styles.TextContainer} ${
 										exists ? styles.Success : ''
 									}`}
 								>
+									<div className={styles.IconContainer}>
+										<img alt={ALT_TEXT.QUESTION_MARK} src={questionMark} />
```

Comment:
this is being used as part of mint flow too. Are these changes intended for both flows?

Reply:
Added a comment to discord moto channel.

https://user-images.githubusercontent.com/39112648/171621098-f832cf9f-ccf2-482e-87ba-f4f051d8fdb3.mp4